boards/actinius-icarus/src/lib.rs

```diff
@@ -0,0 +1,653 @@
+//! Board support crate for the Actinius Icarus -
+//! https://www.actinius.com/get-started
+//!
+//! NB: There is currently no Rust driver for the on-board ST LIS2DH12
```

Comment:
@hannobraun is this the same as the dwm1001 accelerometer?

Reply:
Looks like it. The DWM1001 data sheet say LIS2DH12TR.